backend/ctgov/tests.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+from ctgov.models import SearchStudies
+
+
+# Create your tests here.
+class SearchStudiesTest(APITestCase):
+
+    # Test to check status of post request and results count based on study status
+    def test_completed_studies_count(self):
+        url = "http://localhost:8000/ctgov/api/search_studies"
```

Comment:
How are the tests run currently? Are they run locally or run by a GitHub action? Curious about using localhost within the URL here and if it will cause issues running in GitHub if we decide to go that route.

Reply:
Adding on to this. Can you include documentation on how to run the tests in backend/README?

These would count as integration tests rather than unit tests since they use an actual running instance of the database. This is fine but we need to make sure the unit and integration tests are run with different commands and we need to make sure to document it clearly.